XLS-0033d-CFTs/README.md

```diff
@@ -102,43 +101,37 @@ A set of flags indicating properties or other options associated with this **`CF
-| `lsfCannotFreezeBalances`  | ️`0x0002`  | If set, indicates that _individual_ balances cannot be frozen. This has no effect on the issuers ability to globally freeze, yet provides token holders with more assurance that individual token holders will not be frozen on an ad-hoc basis, but instead all tokens will only ever be frozen or unfrozen together. |
+| `lsfLocked`                | ️`0x0001`  | If set, indicates that all balances are locked. |
+| `lsfCanLock`  | ️`0x0002`  | If set, indicates that the issuer can lock an individual balance or all balances of this CFT.  If not set, the CFT cannot be locked in any way.|
 | `lsfRequiresAuthorization` | ️`0x0004`  | If set, indicates that _individual_ holders must be authorized. This enables issuers to limit who can hold their assets.  |
 | `lsfCanEscrow`             | `0x0008`  | If set, indicates that _individual_ holders can place their balances into an escrow. |
 | `lsfCanTrade`              | `0x0010`  | If set, indicates that _individual_ holders can trade their balances using the XRP Ledger DEX or AMM.
 | `lsfTransferable`          | ️`0x0020`  | If set, indicates that tokens held by non-issuers may be transferred to other accounts. If not set, indicates that tokens held by non-issuers may not be transferred except back to the issuer; this enables use-cases like store credit. |
 | `lsfAllowClawback`         | ️`0x0040`  | If set, indicates that the issuer may use the `Clawback` transaction to clawback value from _individual_ holders.|
 
-With the exception of the `lsfFrozen` flag, which can be mutated via the `**CFTokenIssuanceSet**` transactions, these flags are **immutable**: they can only be set during the **`CFTokenIssuanceCreate`** transaction and cannot be changed later.
+With the exception of the `lsfLocked` flag, which can be mutated via the `**CFTokenIssuanceSet**` transactions, these flags are **immutable**: they can only be set during the **`CFTokenIssuanceCreate`** transaction and cannot be changed later.
 
 ###### 1.2.1.1.2.3. `Issuer`
 
 The address of the account that controls both the issuance amounts and characteristics of a particular fungible token.
 
-###### 1.2.1.1.2.4. `AssetCode`
-
-A 160-bit blob of data. We recommend using only upper-case ASCII letters, ASCII digits 0 through 9, the dot (`.`) character, and the dash (`-`) character. Dots and dashes should never be the first character of an asset code, and should not be repeated sequentially.
-
-While it is possible to store any arbitrary data in this field, implementations that detect the above recommended character conformance can and should display them as human-readable text, allowing issuers to support well-known ISO-4207 currency codes in addition to custom codes. This also helps prevents spoofing attacks where a [homoglyph](https://en.wikipedia.org/wiki/Homoglyph) might be used to trick a person into using the wrong asset code.
-
-###### 1.2.1.1.2.5. `AssetScale`
+###### 1.2.1.1.2.4. `AssetScale`
```

Comment:
Ethereum has the equivalent concept of "Decimals" on chain, and I find some value in keeping that

https://eips.ethereum.org/EIPS/eip-20

Reply:
Not opposed - I also find some value of it on-chain. However, I was thinking it would just be located in `CFTokenMetadata` (but still on-chain).

Are you saying you prefer `AssetScale` _outside_ of metadata?

Reply:
Yeah I am advocating for it be a first-class field outside of metadata, although I can understand the rationale that everything off-chain purpose should live in metadata.  I think it comes down to, "should off-chain fields that we want standardization on live in metadata and hope people adopt, or should we enforce it using a first-class field?"  Either outcome is reasonable to me though, no strong opinion here

Reply:
I think we can leave it as it is for now(having it as a separate field). It is something that is fairly minor and easy to change, so we can make modifications anytime during implementation phase.

Reply:
Sounds good to me to revisit later. Resolving for now, and will open a PR potentially in the future to revisit.